src/components/KonnektSearch.vue

```diff
@@ -57,6 +59,7 @@ class Person {
 }
 export default {
```

Comment:
Line 15 is causing some issues, change that to `v-container(class="v-responsive__sizer v-responsive__content")` if it is required. Although can you explain why this is needed?

Reply:
no it's not needed actually. I was trying to explore v-responsive.

Reply:
removed them